src/librustc_codegen_llvm/attributes.rs

```diff
@@ -263,7 +263,7 @@ pub fn from_fn_attrs(cx: &CodegenCx<'ll, 'tcx>, llfn: &'ll Value, instance: ty::
     // Windows we end up still needing the `uwtable` attribute even if the `-C
     // panic=abort` flag is passed.
     //
-    // You can also find more info on why Windows is whitelisted here in:
+    // You can also find more info on why Windows is allowlisted here in:
```

Comment:
could we not use "allowlist" as a verb? "allow" or "permit" (as in `src/libcore/ffi.rs`) sounds much better

```suggestion
    // You can also find more info on why Windows is allowed here in:
```